.agents/skills/codex-git-guard/SKILL.md

```
@@ -0,0 +1,57 @@
+- Use even if the user only asked for code changes when the task is likely to end in a commit.
+- Do not use for read-only git commands such as `git status`, `git diff`, or `git log`.
+
+## Pipeline
+
+1. Check the current branch first:
+
+```powershell
+git rev-parse --abbrev-ref HEAD
+```
+
+2. If the branch is `develop`, `master`, or `main`, stop. Do not commit or push there.
```

markdownlint-cli2 (0.22.0)

[warning] Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/2/3

(MD029, ol-prefix)